src/GovUk.Education.ExploreEducationStatistics.Admin/Migrations/ContentMigrations/20200825153306_EES1145AllowReleasePreviousVersionIdToBeNullable.cs

```diff
@@ -0,0 +1,52 @@
+﻿using System;
+using Microsoft.EntityFrameworkCore.Migrations;
+
+namespace GovUk.Education.ExploreEducationStatistics.Admin.Migrations.ContentMigrations
+{
+    public partial class EES1145AllowReleasePreviousVersionIdToBeNullable : Migration
+    {
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+            migrationBuilder.AlterColumn<Guid>(
+                name: "PreviousVersionId",
+                table: "Releases",
+                nullable: true,
+                oldClrType: typeof(Guid),
+                oldType: "uniqueidentifier");
+
+            migrationBuilder.UpdateData(
+                table: "Releases",
+                keyColumn: "Id",
+                keyValue: new Guid("4fa4fe8e-9a15-46bb-823f-49bf8e0cdec5"),
+                column: "PreviousVersionId",
+                value: null);
+
+            migrationBuilder.Sql(@"
+                UPDATE Releases SET Releases.PreviousVersionId = null 
+                WHERE Releases.PreviousVersionId = Releases.Id
+                   OR Releases.PreviousVersionId = '00000000-0000-0000-0000-000000000000'");
```

Comment:
With bug https://dfedigital.atlassian.net/browse/EES-1344 we are going to lose track of the Amendments and whether they are a previous version or not, if we run this before we fix that.

Reply:
Actually, not sure there are any Releases with PreviousVersionId equal to 00000000-0000-0000-0000-000000000000 in the Content database, I think they are all in the Statistics databases.